weblate/static/js/vendor/main.js.license

```diff
@@ -0,0 +1,6 @@
+Copyright © Michal Čihař <michal@weblate.org>
```

Comment:
This needs to list all copyright owners:

```suggestion
Copyright © Michal Čihař <michal@weblate.org>
Copyright © OpenJS Foundation and other contributors
```

PS: You should add yourself probably as well.

Reply:
Okay